core/templates/pages/exploration-player-page/learner-experience/conversation-skin.component.spec.ts

```diff
@@ -771,6 +771,8 @@ describe('Conversation skin component', () => {
 
     componentInstance.ngOnInit();
     windowRef.nativeWindow.onresize(null);
+
+    // Tick has been used here to handle flake in frontend.
```

Comment:
I think this needs more explanation in the comment. Ditto below.

Reply:
updated